### PR TITLE
Added relative links

### PR DESCRIPTION
**Pull request by habeeba-naaz**
I have changed specified list of internal links to relative links and tested all changed links...All are working fine. Apart from those specified links,some links which were not specified were not working ,then i modified those links also to relative links and checked as well.Please review my pull request...if any changes to do still let me know.

### CODE REVIEW

**Comment by vegetabill** [comment]
@habeeba-naaz thanks I will review this.

For future reference you should always reference the issue number in your PR: 

#1128

**Comment by habeeba-naaz** [comment]
Okay...sure

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ If you would like to contribute to the curriculum, please [sign up here](https:/
 **Get started:**
 - Choose an issue from the [TO-DO](https://github.com/Techtonica/curriculum/projects/2) column that seems right for you & message Techtonica staff. If you know someone connected to Techtonica, they may be able to add you to our slack channel. If you are here for GSSoC19, message Alina L. (M) and say you'd like to pick up the issue.
 - In the issue, you should find a link for a lesson outline and for corresponding slides.
-- If it is in the ["TO-DO"](https://github.com/Techtonica/curriculum/projects/2) column, that means it is available, even if there's an "assignee".
-- To **create an outline**, fork the repo or commit directly onto the master branch.  [Here is a topic outline template that explains what we need for each section](https://github.com/Techtonica/curriculum/blob/master/_templates/topic-outline.md).
+- If it is in the ["TO-DO"](https://github.com/Techtonica/curriculum/blob/master/projects/2) column, that means it is available, even if there's an "assignee".
```
Comment (vegetabill) [inline CONTRIBUTING.md]: Please undo this line. You broke this link.

File: api/twilio-api.md [hunk 2]
```diff
@@ -52,27 +52,27 @@ TwilioQuest is an interactive tutorial where users can learn how to use Twilio t
 	- **Exercise:** Create a POST request using JSON and look at the Response.
 
 - Example:
-![alt text](https://github.com/Techtonica/curriculum/blob/master/api/Screen%20Shot%202018-10-08%20at%207.15.15%20PM.png)
+![alt text](./Screen%20Shot%202018-10-08%20at%207.15.15%20PM.png)
```
Comment (vegetabill) [inline api/twilio-api.md]: 👍

File: javascript/javascript-6-object-literals.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ About 60 minutes
...
-- [JavaScript 1 Lesson](/javascript/javascript-1.md)
-- [JavaScript 2 Lesson](/javascript/javascript-2.md)
-- [JavaScript 3 Lesson](/javascript/javascript-3.md)
-- [JavaScript 4 Lesson](/javascript/javascript-4.md)
-- [JavaScript 5 Lesson](/javascript/javascript-5.md)
+- [JavaScript 1 Lesson](../javascript/javascript-1.md)
+- [JavaScript 2 Lesson](../javascript/javascript-2.md)
+- [JavaScript 3 Lesson](../javascript/javascript-3.md)
+- [JavaScript 4 Lesson](../javascript/javascript-4.md)
+- [JavaScript 5 Lesson](../javascript/javascript-5.md)
```
Comment (vegetabill) [inline javascript/javascript-6-object-literals.md]: Please leave these unchanged. We only care about links that specify the full URL. Sorry if that wasn't clear in the issue text.

File: projects/data-structures-algorithms-sprint.md [hunk 4]
```diff
@@ -6,14 +6,14 @@ This sprint will also help you practice study skills, summarizing, teamwork, and
...
-- [Trees](/data-structures/trees.md) 
-- [Searching](/algorithms/searching.md) and [Sorting](/master/algorithms/sorting.md) Algorithms 
+- [Data Structures](../data-structures/intro-to-data-structures.md) 
+- [Stacks](../data-structures/stack.md) 
+- [Queues](../data-structures/queues.md) 
+- [Deques](../data-structures/deque.md) 
+- [Linked Lists](../data-structures/linked-lists.md) 
+- [Hash Tables](../data-structures/hash-tables.md) 
+- [Trees](../data-structures/trees.md) 
+- [Searching](../algorithms/searching.md) and [Sorting](../algorithms/sorting.md) Algorithms
```
Comment (vegetabill) [inline projects/data-structures-algorithms-sprint.md]: Please leave these unchanged.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -5,11 +5,11 @@ If you would like to contribute to the curriculum, please [sign up here](https:/
 **Get started:**
 - Choose an issue from the [TO-DO](https://github.com/Techtonica/curriculum/projects/2) column that seems right for you & message Techtonica staff. If you know someone connected to Techtonica, they may be able to add you to our slack channel. If you are here for GSSoC19, message Alina L. (M) and say you'd like to pick up the issue.
```
Comment (Kundan28) [inline CONTRIBUTING.md]: @vegetabill Don't you think GSSoC19 could be changed to GSSoC20 ?

File: CONTRIBUTING.md (hunk 1 above)
Comment (habeeba-naaz) [reply CONTRIBUTING.md]: which link i broke ..please mention link?

File: javascript/javascript-6-object-literals.md (hunk 3 above)
Comment (habeeba-naaz) [reply javascript/javascript-6-object-literals.md]: we i kept  unchanged those links ..those links are not working is it okay?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Kundan28) [reply CONTRIBUTING.md]: I think he's talking about `https://github.com/Techtonica/curriculum/blob/master/projects/2`

File: projects/data-structures-algorithms-sprint.md (hunk 4 above)
Comment (habeeba-naaz) [reply projects/data-structures-algorithms-sprint.md]: If i unchanged these links... those links are not working...is this okay?

File: javascript/javascript-6-object-literals.md (hunk 3 above)
Comment (Kundan28) [reply javascript/javascript-6-object-literals.md]: If the link are not working then I think better to keep that changes.

File: api/twilio-api.md (hunk 2 above)
Comment (habeeba-naaz) [reply api/twilio-api.md]: what should i do in api/twillo-api.md?

File: javascript/javascript-6-object-literals.md (hunk 3 above)
Comment (habeeba-naaz) [reply javascript/javascript-6-object-literals.md]: Please check CONTRIBUTING.md  where link is broken ?

File: CONTRIBUTING.md (hunk 1 above)
Comment (habeeba-naaz) [reply CONTRIBUTING.md]: yes i got it .. thank you.I changed, Now it's working.So i only corrected CONTRIBUTING.md .ready to send pull request

Comment (habeeba-naaz) [reply CONTRIBUTING.md]: I changed,commited, and again pushed but unable to see any PR request?

File: projects/data-structures-algorithms-sprint.md (hunk 4 above)
Comment (vegetabill) [reply projects/data-structures-algorithms-sprint.md]: Oh I see what you mean. The correct fix, I believe, is to remove the leading `/` then. Thanks for catching that!

File: CONTRIBUTING.md (hunk 5 above)
Comment (vegetabill) [reply CONTRIBUTING.md]: @Kundan28 there is a whole section above so we should actually remove that sentence. 

@habeeba-naaz please remove the "If you are here..." sentence please.

File: javascript/javascript-6-object-literals.md (hunk 3 above)
Comment (vegetabill) [reply javascript/javascript-6-object-literals.md]: @Kundan28 it's not that I want them broken but it's illogical for a link to go up one level from the directory it's in just to go back down.

These should be `./javascript-1.md` etc

File: api/twilio-api.md (hunk 2 above)
Comment (vegetabill) [reply api/twilio-api.md]: Nothing just +1 this find. I forgot about the images.

File: CONTRIBUTING.md (hunk 5 above)
Comment (habeeba-naaz) [reply CONTRIBUTING.md]: I changed

File: projects/data-structures-algorithms-sprint.md (hunk 4 above)
Comment (habeeba-naaz) [reply projects/data-structures-algorithms-sprint.md]: If we keep leading slash also not working, so i left as it is like 
../data-structures/stack.md

File: javascript/javascript-6-object-literals.md (hunk 3 above)
Comment (habeeba-naaz) [reply javascript/javascript-6-object-literals.md]: This i changed as you specified

**Comment by habeeba-naaz** [comment]
Still any changes to make?

**Comment by habeeba-naaz** [comment]
closes issue #1128

**Comment by Kundan28** [comment]
Fixes #1128

**Comment by habeeba-naaz** [comment]
Thank you for merging

**Comment by Kundan28** [comment]
@vegetabill Please close this issue. As it's already completed.

**Pull request merged**